<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PivotMain</class>
 <widget class="QWidget" name="PivotMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>611</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Drag from Labels and drop in the desired field</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Labels</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Labels"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Select Option</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="selectOption">
             <item>
              <property name="text">
               <string>sum</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>average</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>max</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string comment="minimum">min</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>count</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="TotalRows">
             <property name="text">
              <string>Total Rows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="TotalColumns">
             <property name="text">
              <string>Total Columns</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Rows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Rows"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Columns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Columns"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Values</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Values"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
